rust/macros/module.rs

```diff
@@ -300,6 +300,25 @@ impl ModuleInfo {
 
         info
     }
+
+    /// Parse ["First", "Second"] into Some(vec!["First", "Second"])
+    fn parse_authors(it: &mut token_stream::IntoIter) -> Vec<String> {
+        let mut authors: Vec<String> = vec![];
+        let group = expect_group(it);
+        assert_eq!(group.delimiter(), Delimiter::Bracket);
+        let mut stream = group.stream().into_iter();
+        loop {
+            if let Some(s) = try_string(&mut stream) {
```

Comment:
We can probably `expect_string` here. Otherwise we'd allow extra commas without any authors in between.

Reply:
Fixed, see new commit.